ArduPlane/Plane.h

```diff
@@ -1003,6 +1003,8 @@ class Plane : public AP_Vehicle {
     bool verify_nav_wp(const AP_Mission::Mission_Command& cmd);
 #if HAL_QUADPLANE_ENABLED
     bool verify_landing_vtol_approach(const AP_Mission::Mission_Command& cmd);
+    // vtol takeoff from AP_Vehicle for quadplane.
+    bool start_takeoff(const float alt) override;
```

Comment:
The method as implemented doesn't need to be limited to QUADPLANE_ENABLED.

Reply:
For fixed wing takeoff, we already recommend using TAKEOFF mode via the ModeSwitch service, for now, and everyone seems happy with it.